content/agent/basic_agent_usage/docker.md

```diff
@@ -40,27 +40,27 @@ docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
 
 ### Environment variables
 
-The agent is highly customizable, here are the most used environment variables:
+The Agent is highly customizable, here are the most used environment variables:
 
 #### Global options
 
 - `DD_API_KEY`: your API key (**required**)
 - `DD_HOSTNAME`: hostname to use for metrics (if autodetection fails)
 - `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
 
-#### Optional collection agents
+#### Optional collection Agents
 
 These features are disabled by default for security or performance reasons, you need to explicitly enable them:
 
-- `DD_APM_ENABLED`: run the trace-agent along with the infrastructure agent, allowing the container to accept traces on `8126/tcp`
-- `DD_LOGS_ENABLED`: run the [log-agent][3] along with the infrastructure agent.
+- `DD_APM_ENABLED`: run the trace-agent along with the infrastructure Agent, allowing the container to accept traces on `8126/tcp`
+- `DD_LOGS_ENABLED`: run the [log-agent][3] along with the infrastructure Agent.
 - `DD_PROCESS_AGENT_ENABLED`: enable live process collection in the [process-agent][4]. The Live Container View is already enabled by default if the Docker socket is available
 
-#### Dogstatsd (custom metrics)
+#### DogStatsD (custom metrics)
 
 Send custom metrics via [the statsd protocol][5]:
 
-- `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`: listen to dogstatsd packets from other containers, required to send custom metrics
+- `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`: listen to DogStatsD packets from other containers, required to send custom metrics
```

Comment:
DogStatsD when we're referring to the concept or functionality; dogstatsd when we're referring to the literal executable.

And yes, I am highly nit-picking here. 😉